packages/attestation-service/config/.env.development

```diff
@@ -1,18 +1,26 @@
 DATABASE_URL=sqlite://db/dev.db
 CELO_PROVIDER=http://localhost:8545
-CELO_VALIDATOR_ADDRESS=0xE6e53b5fc2e18F51781f14a3ce5E7FD468247a15
+CELO_VALIDATOR_ADDRESS=x
 ATTESTATION_SIGNER_ADDRESS=x
 APP_SIGNATURE=x
-SMS_PROVIDERS=twilio
+SMS_PROVIDERS=nexmo,twilio
+SMS_PROVIDERS_US=twilio,nexmo
+SMS_PROVIDERS_MX=nexmo
```

Comment:
Wonder if it'd be more convenient to pass in a JSON string with a country code to provider map instead of per-country overrides which could stack up

Reply:
I considered that but these are env vars and the escaping etc would get messy --- short of changing the structure significantly